zubhub_frontend/zubhub/src/components/protected_route/ProtectedRoute.jsx

```diff
@@ -1,25 +1,10 @@
 import React from 'react'
-import { Redirect, Route } from 'react-router-dom'
-import PageWrapper from '../../views/PageWrapper';
-import LoadingPage from '../../views/loading/LoadingPage';
+import { Navigate } from 'react-router-dom';
 
-const ProtectedRoute = ({ component: LazyComponent, ...props }) => {
-    return (
-      <Route
-        {...props}
-        render={routeProps =>
-          props.auth?.token ? (
-            <PageWrapper {...props} {...routeProps}>
-              <React.Suspense fallback={<LoadingPage />}>
-                <LazyComponent {...routeProps} {...props} />
-              </React.Suspense>
-            </PageWrapper>
-          ) : (
-            <Redirect to="/login" />
-          )
-        }
-      />
-    );
+const ProtectedRoute = props => {
```

Comment:
no idea why we introduced this change. To the person who introduced it, why are we making this change?
If it's not breaking anything then we should revert it back

(mentioning both @brrkrmn and @coderatomy so the person we introduced this change will get pinged by github)

Reply:
This is a breaking change when it comes to react-router-dom v6.